docs/applications/server.md

```diff
@@ -204,3 +206,19 @@ For more information on how to use our built in `cli` to interact with the serve
     - `psbt` - PSBT serialized in hex or base64 format
  - `converttopsbt` `unsignedTx` - Creates an empty psbt from the given transaction
     - `unsignedTx` - serialized unsigned transaction in hex
+
+
+### Sign PSBT with Wallet Example
```

Comment:
Hmm, maybe move this to `psbt.md`?

Reply:
`psbt.md` shows how to use them within the library, nothing to do with the wallet or server